src/libponyrt/platform/threads.c

```diff
@@ -3,15 +3,17 @@
 #endif
 #include <platform.h>
 
-#if defined(PLATFORM_IS_LINUX) || defined(PLATFORM_IS_FREEBSD)
+#if defined(PLATFORM_IS_LINUX) || defined(PLATFORM_IS_BSD)
 #include <sched.h>
 #include <sys/time.h>
 #include <sys/resource.h>
 #include <sys/mman.h>
 #endif
 
-#if defined(PLATFORM_IS_FREEBSD)
+#if defined(PLATFORM_IS_BSD)
 #include <pthread_np.h>
```

Comment:
i noted that we are inconsistent with when we indent includes (see atomics.h which is rather different)